README.md

```diff
@@ -483,102 +492,128 @@ network_connections:
-and later enabling NetworkManager is not guaranteed to work automatically. Possibly
+The `network` role supports the same configuration scheme for both providers (`nm`
+and `initscripts`). That means, you can use the same playbook with NetworkManager
+and initscripts. However, note that not every option is handled exactly the same
+by every provider. Do a test run first with `--check`.
+
+It is not supported to create a configuration for one provider, and expect another
+provider to handle them. For example, creating profiles with the `initscripts` provider,
+and later enabling NetworkManager is not guaranteed to work automatically. Possibly,
 you have to adjust the configuration so that it can be used by another provider.
 
-For example what will work is to configure a RHEL6 host with initscripts and upgrade to
-RHEL7 while continuing to use initscripts on RHEL7. What is not guaranteed to work
-it to upgrade to RHEL7, disable initscripts and expect NetworkManager to take over
-the configuration automatically.
+For example, configuring a RHEL6 host with initscripts and upgrading to
+RHEL7 while continuing to use initscripts in RHEL7 is an acceptable scenario. What
+is not guaranteed is to upgrade to RHEL7, disable initscripts and expect NetworkManager
+to take over the configuration automatically.
 
 Depending on NetworkManager's configuration, connections may be stored as ifcfg files
-as well, but again it is not guaranteed that plain initscripts can handle these ifcfg files
+as well, but it is not guaranteed that plain initscripts can handle these ifcfg files
 after disabling the NetworkManager service.
+
+Limitations
+-----------
+
+Ansible usually works via the network, for example via SSH. There are some limitations to be considered:
+
+The `network` role does not  support how to bootstrap networking configuration. One
+option may be [ansible-pull](https://docs.ansible.com/ansible/playbooks_intro.html#ansible-pull).
+Another option maybe be to initially auto-configure the host during installation
+(ISO based, kickstart, etc.), so that the host is connected to a management LAN
+or VLAN. It strongly depends on your environment.
+
+For `initscripts` provider, deploying a profile merely means to create the ifcfg
+files. Nothing happens automatically until the play issues `ifup` or `ifdown`
+via the `up` or `down` [states](#state) -- unless there are other
+components that rely on the ifcfg files and react on changes.
+
+The `initscripts` provider requires the different profiles to be in the right
+order when they depend on each other. For example the bonding master device
+needs to be specified before the slave devices.
+
+When removing a profile for NetworkManager it also takes the connection
+down and possibly removes virtual interfaces. With the `initscripts` provider
+removing a profile does not change its current runtime state (this is a future
+deployment for NetworkManager as well).
```

Comment:
maybe use "future feature" instead of "future deployment". Deployments sounds wrong in this context.